lib/models/workflow/workflow.models.ts

```diff
@@ -76,6 +76,9 @@ export namespace WorkflowModels {
             codename?: string;
         };
         step_identifier: SharedContracts.IReferenceObjectContract;
+        due_date?: LanguageVariantModels.ILanguageVariantDueDate;
```

Comment:
![image](https://github.com/user-attachments/assets/5108ede9-34bd-4d4b-9270-fa6aa7eadd30)
I don't see due_date, contributors and notes in MAPI documentation :O

Reply:
Yeah, don't know why it's not there and it confused me as well. 

I was following the PR for .NET SDK where these properties are https://github.com/Kontent-ai-Learn/kontent-ai-learn-code-samples/pull/95/files#diff-76b77d979883e243c45457e13aacee925493416eb532e75fb1b9c9b57f8bdfb7